sdk/src/Core/Amazon.Runtime/Internal/Util/IniFile.cs

```diff
@@ -595,6 +595,9 @@ private bool SeekProperty(ref int lineNumber, out string propertyName, out strin
                 nestedProperty = null;
                 if (TryParseProperty(Lines[lineNumber], out propertyName, out propertyValue))
                 {
+                    // Property keys are considered case-insensitive. Property keys SHOULD be stored lower-cased after parsing.
+                    propertyName = propertyName.ToLower();
```

Comment:
Since this is a file in the customer's environment, shouldn't you use [ToLowerInvariant](https://learn.microsoft.com/en-us/dotnet/api/system.string.tolowerinvariant) here? (I also know we're not 100% consistent within the SDK, I see both methods being used in other places)